src/backend/sgx/thread.rs

```diff
@@ -41,7 +42,8 @@ pub struct Thread {
 
 impl Drop for Thread {
     fn drop(&mut self) {
-        self.keep.tcs.write().unwrap().push(self.tcs)
+        trace!("Dropping thread");
+        // self.keep.tcs.write().unwrap().push(self.tcs)
```

Comment:
What's happening here? :thinking:

Reply:
read the commit message for this chunk

Reply:
Does this comment represent a FIXME? If so, it should mention that explicitly.

Reply:
added comment and issue https://github.com/enarx/enarx/issues/2200

Reply:
pushed a fixup